usaspending_api/references/models.py

```diff
@@ -367,7 +367,7 @@ def load_city_county_data(self):
 
 class LegalEntity(DataSourceTrackedModel):
     legal_entity_id = models.BigAutoField(primary_key=True, db_index=True)
-    recipient_name = models.TextField(blank=True, verbose_name="Recipient Name", null=True)
+    recipient_name = models.TextField(blank=True, verbose_name="Recipient Name", null=True, db_index=True)
```

Comment:
Current index only used 526 times compared to the 13160 times for the next-most used (and 1724359751 using the PK index)

It might be better to drop the DB index instead of defining the index here.